### DISCUSSION TITLE
Oracle performance

### DISCUSSION THREAD

**Discussion by cherfim** [dbeaver/dbeaver#16774] [General]
Hello,

I was wondering if anyone is having issues with Oracle DB performance ? I created [this issue](https://github.com/dbeaver/dbeaver/issues/16664) but it was not answered. anyone in the same boat as I ?

**Comment by roboteer** [comment]
As a decade long SQLDeveloper user and 1-week long DBeaver user, I can tell that fetching data does feel slightly slower in DBeaver, but nowhere as drastic as your experience. More like +2 seconds (Oracle 12g). Everything else (metadata etc) on the other hand feels much more responsive and generally awesome. So far looks like I'm converted. 

Okay, I did ran some parallel comparisons: 
1) With 300k rows of simpler data DBeaver was actually slightly faster! So subjectivity is not to be trusted.
2) But 300k rows with some CLOBs, DBeaver took leisurely 5 minutes, while SQLDeveloper was done in 18 seconds.. 

So yeah, there is yet some room for improvements. 
This is different from your issue though, since a) i guess you did not have CLOBs, 2) "Export from Query" took just as long, if not longer.
But thanks for pointing this out, something to keep an eye on.
